addons/monit/monit_checks_configurations/00_packetfence.tt

```diff
@@ -55,12 +55,6 @@ CHECK PROCESS packetfence-httpd.admin MATCHING "pf/var/conf/httpd.conf.d/httpd.a
     stop program  = "/usr/local/pf/bin/pfcmd service httpd.admin stop"
     if 3 restarts within 10 cycles then alert
 
-CHECK PROCESS packetfence-httpd.graphite MATCHING "pf/var/conf/httpd.conf.d/httpd.graphite"
```

Comment:
Shouldn't we monitor the new netdata services ?

Reply:
Done